modules/adv-install/pages/response-files.adoc

```diff
@@ -1,3 +1,136 @@
+
+You can use this XML file, along with a `variables` file (typically with the same name as the XML file but with a file extension of `.variables`, for example `myinstall.variables`), to provision JBoss EAP instances on other servers in an *automated* manner (non-interactive, unattended installation). This approach gives you flexibility to roll out JBoss EAP instances across multiple servers using wrapper shell scripts, configuration management tools like Ansible, Terraform, Puppet, and more.
+
+TIP: You can version control the automated installer XML file and the `.variables` file in Git and optionally use GitOps and Infrastructure as Code (IaC) techniques to roll out JBoss EAP instances in your development, staging, QA and Production environments.
+
+[TIP]
+====
+You need not copy the installer XML files and variables file to every system. You can serve these files over http or ftp and pass the location to the installer JAR as arguments:
+
+$ *java -jar installer.jar http://myhost/myinstall.xml -variablesfile http://myhost/myinstall.xml.variables*
+====
+
+WARNING: The automated installer is not backward compatible! You cannot use an installation script generated from a previous version of JBoss EAP to install a new version. Only use installation scripts generated by the same version of JBoss EAP.
+
+== References
+
+* https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html-single/installation_guide/index#automated-installer-installation_default[Using the automated installer installation]
+
+== Lab: Install JBoss EAP using the Automated Installer
+
+=== Pre-requisites
+
+* You should have previously installed JBoss EAP using the JAR installer and saved the installation options into an automated installer XML file.
+
+* Alternatively, you can download sample https://raw.githubusercontent.com/RedHatQuickCourses/eap-qc-apps/main/admin1/eap-automated/myinstall.xml[myinstall.xml] and https://raw.githubusercontent.com/RedHatQuickCourses/eap-qc-apps/main/admin1/eap-automated/myinstall.xml.variables[myinstall.xml.variables] files, and copy it to the machine where you want to install JBoss EAP
+
+=== Steps
+
+. If you have a `myinstall.xml` automated installation file from a previous GUI installation, then you can use it in this lab. Ensure that you edit the `myinstall.xml.variables` file and enter the password for the `admin` user, otherwise the installer will prompt you for the password.
+
+. If you download the sample https://raw.githubusercontent.com/RedHatQuickCourses/eap-qc-apps/main/admin1/eap-automated/myinstall.xml[myinstall.xml] and https://raw.githubusercontent.com/RedHatQuickCourses/eap-qc-apps/main/admin1/eap-automated/myinstall.xml.variables[myinstall.xml.variables] files, then you need to edit both files and change some values before you proceed with the installation.
```

Comment:
Should we mention about what values might need to be changes in myinstall.xml? Like I checked my file and it shows:

$ cat /opt/EAP-8.0.0/myinstall.xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<installation-data xmlns="urn:jboss:installer:record:1.0.0">
    <admin-user username="admin"/>
    <target-directory>/opt/EAP-8.0.0</target-directory>
    <maven-repositories>
        <maven-repository repository-id="jboss-eap-8.0-product-repository" repository-url="https://maven.repository.redhat.com/ga/"/>
    </maven-repositories>
    <selected-packages>
        <package name="component_selection.eap"/>
        <package name="component_selection.appclient"/>
        <package name="component_selection.docs"/>
        <package name="component_selection.modules"/>
        <package name="component_selection.welcome"/>
    </selected-packages>
    <post-install-tasks>
        <post-install-task name="add-admin"/>
    </post-install-tasks>
</installation-data>

So, in this case only target-directory might need to be changed.

Similarly, for myinstall.xml.variables, only adminPassword need to be update.

$ cat /opt/EAP-8.0.0/myinstall.xml.variables 
adminPassword=redhat123